cypress/utils/utils.ts

```diff
@@ -1915,6 +1915,15 @@ export function selectColumns(selectedColumns: string[], buttonText: string = sa
         });
 }
 
+export function selectLogView(logName) {
```

Comment:
Please add the type of the `logName` parameter and `void` as the return type

Reply:
done